third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -2628,10 +2634,10 @@ StackingContext.prototype = {
     left = left || 0;
     top = top || 0;
     opacity = opacity || 1;
-    transform = transform || {};
-    if (this.transform.scale !== undefined) {
-      transform.scale = (transform.scale || 1) * this.transform.scale;
-    }
+    transform = transform || DefaultTransform;
+
+    // TODO: handle scaleX/Y/Z
+    transform.scale = getContextTransformScale(transform) * getContextTransformScale(this.transform);
```

Comment:
You shouldn't mutate `transform` particularly if you're falling back to `DefaultTransform`